applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -26,7 +26,7 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_BOND_LOG_LEVEL);
 #define BT_ID_LUT_STORAGE_NAME		"bt_lut"
 
 #define ON_START_CLICK(CLICK)		(CLICK + CLICK_COUNT)
-#define ON_START_CLICK_UPTIME_MAX	K_SECONDS(6)
+#define ON_START_CLICK_UPTIME_MAX	(6 * MSEC_PER_SEC)
```

Comment:
this seems unneeded; at ble_latency.c you use the API that was used here.

Reply:
K_SECONDS converts to struct k_timeout_t, but ON_START_CLICK_UPTIME_MAX is used in comparison expressions, so it must remain an integer value.